src/utils/helpers/sanatizeText.js

```diff
@@ -0,0 +1,5 @@
+export const sanatizeText = (text) => {
+  text = text.trim();
+  text = text.replaceAll(/<\/?[^>]+(>|$)/gi, "");
+  return text;
```

Comment:
Could we set it to one go ?
```suggestion
  return text.trim().replaceAll(/<\/?[^>]+(>|$)/gi, "");
```